pkgs/os-specific/darwin/apple-sdk/default.nix

```diff
@@ -107,8 +107,8 @@ let
         fi
 
         local tbd_source=${darwin-stubs}/System/Library/Frameworks/$nested_path/Versions/$current
-        if [ "${name}" != "Kernel" ]; then
-          # The Kernel.framework has headers but no actual library component.
+        # Skip frameworks which have no generated tbd file.
+        if stat --printf="" "$tbd_source/*.tbd" 2>/dev/null; then
```

Comment:
> The tbd file should be at $(nix-build -A darwin.apple_sdk.frameworks.Foundation)/Library/Frameworks/Foundation.framework/Foundation.tbd, but it doesn't exist on this branch.

I found the issue about this and fixed it. 


The change is required until https://github.com/NixOS/darwin-stubs/pull/2 is merged or otherwise I'll get
```
copying path '/nix/store/akqlhgsf02vzhw330nrbd7hrzdh11i68-apple-framework-ScriptingBridge' from 'https://cache.nixos.org'...
copying path '/nix/store/akw65s6c62zx5xwi4pcy76jnavgxpm9j-apple-framework-ContactsPersistence' from 'https://cache.nixos.org'...
copying path '/nix/store/mbh30nci66wagn19zsyz9vc1mym3mfw9-apple-framework-JavaScriptCore' from 'https://cache.nixos.org'...
building '/nix/store/7lhnl21i31lmkkp9kgh7sn8bh2pnvmpd-apple-framework-Contacts.drv'...
patching sources
configuring
no configure script, doing nothing
building
no Makefile, doing nothing
installing
cp: missing destination file operand after '.'
Try 'cp --help' for more information.
builder for '/nix/store/7lhnl21i31lmkkp9kgh7sn8bh2pnvmpd-apple-framework-Contacts.drv' failed with exit code 1
building '/nix/store/v0a0v5ilbn33hxqrly8bgcd5fg9hicb3-apple-framework-CoreVideo.drv'...
cannot build derivation '/nix/store/vz957sq4ka380j1n3sfq9f35bzzq691h-python3.8-pyobjc-framework-Contacts-6.2.2.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/89wpq0spvff4qvk6f3hpslb9lbnfdw9s-python3.8-pyobjc-framework-ContactsUI-6.2.2.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/6vwlfy26xxjn5sfnazw67262h99f3pww-python3.8-pyobjc-6.2.2.drv': 1 dependencies couldn't be built
error: build of '/nix/store/6vwlfy26xxjn5sfnazw67262h99f3pww-python3.8-pyobjc-6.2.2.drv' failed
```